ckanext/aircan_connector/action.py

```diff
@@ -27,17 +27,25 @@
                         ' See <a href="https://github.com/datopian/ckanext-aircan#airflow-instance-on-google-composer"> Airflow instance on Google Composer </a>' \
                         ' section in AirCan docs for more.'
 
-def datapusher_submit(context, data_dict):
+
+def aircan_submit(context, aircan_data_dict):
+    log.info("Aircan submit action")
+    data_dict = get_action('resource_create')(context, aircan_data_dict)
```

Comment:
@hannelita will not work... All this process here is happened **after** `resource_create` actually.
I've already called `resource_create/update` from CKAN portal vis GUI or via API directly at this point.

Reply:
ah ok
I called it because weirdly the dict was not coming with the parameters I needed